tests/SessionTest.php

```diff
@@ -83,34 +101,10 @@ public function testRestart()
         $this->session->restart();
     }
 
-    public function testVisitWithoutRunningSession()
```

Comment:
this test must not be removed. We still need to cover the case of calling ``visit`` on a started and on a non-started session IMO (thus, your mocks don't describe the actual interaction happening, as you removed the mocking of ``isStarted``